python/ray/dashboard/modules/metrics/dashboards/data_dashboard_panels.py

```diff
@@ -114,27 +114,27 @@
     ),
     Panel(
         id=7,
-        title="Bytes Outputted",
-        description="Total bytes outputted by dataset operators.",
-        unit="bytes",
+        title="Bytes Output / Second",
+        description="Bytes output per second by dataset operators.",
+        unit="bytes/sec",
```

Comment:
for `unit`, let's use: `"bytes/sec(SI)"`. this will make sure the auto unit-conversion from Grafana works as intended.

<img width="1789" alt="Screenshot at Aug 22 16-11-14" src="https://github.com/user-attachments/assets/cd813a67-12c7-4e35-8d19-fd3e2bc68c22">

Reply:
after making this change, could you also share a screenshot of the dashboard with the fixes, to verify that the units properly show up on the axes + the conversion works (e.g. once we start hitting 100's of MBs, this should display as MB/s instead of B/s)

Reply:
Updated it to `Bps` because that is what Grafana expects for this unit (rather than the full `bytes/sec(SI)`).

<img width="2690" alt="image" src="https://github.com/user-attachments/assets/86928363-ae2e-4697-9aaa-fcdb9ae4e0bb">